accounts/admin.py

```diff
@@ -32,18 +32,28 @@ class UserAdmin(admin.ModelAdmin):
     ordering = ("username",)
     fieldsets = (
         (None, {"fields": ("username", "pennid")}),
-        (("Personal info"), {"fields": ("first_name", "last_name", "email")}),
+        (("Personal info"), {"fields": ("preferred_name", "first_name", "last_name", "email")},),
```

Comment:
We should probably keep first name in a read-only field just so we can refer to it if needed

Reply:
Moved last_name to read_only as well